src/lib/message/L1ToL2Message.ts

```diff
@@ -237,6 +249,53 @@ export class L1ToL2MessageReader extends L1ToL2Message {
     return currentTimestamp.gte(timeoutTimestamp)
   }
 
+  public async getMessageInputs(): Promise<L1toL2MessageInputs> {
```

Comment:
Some docs here. Also maybe call it getInputComponents() or getInputs(), since we're already on the message